hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -157,15 +159,49 @@ def setUp(self):
 
         s.set_elements(elements)
         s.add_lines(xray_lines)
+        s.axes_manager[0].scale = 0.5
+        s.axes_manager[1].scale = 0.5
         self.signal = s
 
     def test_quant_lorimer(self):
         s = self.signal
+        method = 'CL'
         kfactors = [1, 2.0009344042484134]
+        composition_units = 'weight'
         intensities = s.get_lines_intensity()
-        res = s.quantification(intensities, kfactors)
-        nt.assert_true(np.allclose(res[0].data, np.array(
-                       [22.70779, 22.70779]), atol=1e-3))
+        res = s.quantification(intensities, method, kfactors, composition_units)
+        np.testing.assert_allclose(res[0].data, np.array([[22.70779, 22.70779],
+                    [22.70779, 22.70779]]), atol=1e-3)
+
+    def test_quant_zeta(self):
+        s = self.signal
+        method = 'zeta'
+        compositions_units = 'weight'
+        factors = [20, 50]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors, compositions_units)
+        np.testing.assert_allclose(res[1].data, np.array(
+                [[ 2.7125736e-03,   2.7125736e-03],
+                [  2.7125736e-03,   2.7125736e-03]]), atol=1e-3)
+        np.testing.assert_allclose(res[0][1].data, np.array(
+                [[ 80.962287987,   80.962287987],
+                [  80.962287987,   80.962287987]]), atol=1e-3)
+
+    def test_quant_cross_section(self):
+        s =self.signal
+        method = 'cross_section'
+        factors = [3, 5]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors)
+        np.testing.assert_allclose(res[1][0].data, np.array(
+            [[ 21517.1647074,  21517.1647074],
+            [  21517.1647074,  21517.1647074]]), atol=1e-3)
+        np.testing.assert_allclose(res[1][1].data, np.array(
+           [[ 21961.616621,  21961.616621],
+           [  21961.616621,  21961.616621]]), atol=1e-3)
```

Comment:
Ok - that's fine - have you checked the analysis on a dataset that you've also done by other means? So long as you're very confident that the result here is right, then I suppose it doesn't matter, but it might be nice to also make it representative of the kind of results someone might expect to get in normal usage. Could you crop a very small region e.g. 2x2 pixels of a dataset you know, or can synthesise and then use that, just so it looks sensible?